packages/components/src/DataViewer/DataViewer.stories.js

```diff
@@ -29,67 +47,44 @@ stories
 		</div>
 	))
 	.add('DataTree with highlight', () => {
+		const [jsonPathSelection, setJsonPathSelection] = useState("$['category']");
```

Comment:
This story had multiple issues
- it was not working, on model click to highlight, it doesn't rerender, so the record highlight was not updated. I added a component state to manage the jsonPath to highlight
- it uses cmf, to provide some values but this is a component, not related to cmf. I removed that.